exporter/clickhouseexporter/factory.go

```diff
@@ -119,3 +120,23 @@ func createMetricExporter(
 		exporterhelper.WithRetry(c.RetrySettings),
 	)
 }
+
+func GenerateTTLExpr(ttlDays uint, ttl time.Duration) string {
```

Comment:
Should not be exported
```suggestion
func generateTTLExpr(ttlDays uint, ttl time.Duration) string {
```